File: inventario_visual/Models/MovimientoInventario.vb
Namespace inventario_visual
    Public Class MovimientoInventario
        Public Property Id As Long
        Public Property ProductoId As Long
        Public Property ProveedorId As Long?
        Public Property Precio As Double
        Public Property PrecioBalance As Double
        Public Property Cantidad As Double
        Public Property TipoMovimiento As String
        Public Property FechaMovimiento As DateTime
        Public Property Motivo As String
        Public Property ProductoNombre As String
        Public Property ProveedorNombre As String
    End Class
End Namespace

File: inventario_python/models/movimiento_inventario.py
class MovimientoInventario:
    def __init__(self):
        self.id = 0
        self.producto_id = 0
        self.proveedor_id = None
        self.precio = 0.0
        self.precio_balance = 0.0
        self.cantidad = 0.0
        self.tipo_movimiento = ""
        self.fecha_movimiento = None
        self.motivo = ""
        self.producto_nombre = ""
        self.proveedor_nombre = ""
